lib/compat/wordpress-6.5/block-bindings/sources/pattern.php

```diff
@@ -29,7 +29,9 @@
 	};
 	wp_block_bindings_register_source(
 		'pattern_attributes',
-		__( 'Pattern Attributes', 'gutenberg' ),
-		$pattern_source_callback
+		array(
```

Comment:
There should be no textdomain used for translations expected to land in core.

Reply:
Thanks! I removed it for both the `pattern` and `post-meta` sources in c9a245e0be4dde85535535b84441c9a6914c00b9